### PR TITLE
UNI-540 Better log info and checks

### PR DESCRIPTION
**Pull request by marionleborgne**
* suppress misleading error message that happens even if uninstall was sucessfull
* make sure that unicorn backend was successfully uninstalled & installed

### CODE REVIEW

File: unicorn/app/package.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
   ],
   "scripts": {
     "postinstall": "npm run install:backend",
-    "install:backend": "npm run clean:backend && python ../py/setup.py install",
-    "clean:backend": "python -m pip uninstall unicorn_backend -y || true"
+    "install:backend": "npm run clean:backend && python ../py/setup.py install && python -c 'import unicorn_backend'",
+    "clean:backend": "python -m pip uninstall unicorn_backend -y 2> /dev/null || true && if python -c 'import unicorn_backend'; then exit 1; else exit 0; fi"
```
Comment (lscheinkman) [inline unicorn/app/package.json]: @marionleborgne `/dev/null` will not work on windows

Comment (oxtopus) [inline unicorn/app/package.json]: How about:

```
python -m pip uninstall unicorn_backend -yqqq
```

Without the stream redirection.

**Comment by oxtopus** [comment]
@marionleborgne can you check in on this PR?  It's old, and no longer in a mergeable state.

**Comment by marionleborgne** [comment]
@oxtopus I would close this PR. The `clean:backend` works and is cross platform https://github.com/numenta/numenta-apps/blob/master/unicorn/package.json#L27

**Pull request closed without merging**